internal/controller/thanosruler_controller_test.go

```diff
@@ -26,8 +26,10 @@ import (
 	. "github.com/onsi/gomega"
 
 	monitoringthanosiov1alpha1 "github.com/thanos-community/thanos-operator/api/v1alpha1"
+	"github.com/thanos-community/thanos-operator/internal/pkg/manifests"
 	"github.com/thanos-community/thanos-operator/test/utils"
 
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
nit: group as per https://google.github.io/styleguide/go/best-practices.html#import-ordering

Reply:
I added a -local flag to goimport to prevent this. Hopefully this can be done automatically now :)